ocs_ci/ocs/constants.py

```diff
@@ -915,6 +915,7 @@
 # Managed service deployment constants
 OSD_DEPLOYER = "ocs-osd-deployer"
 OSE_PROMETHEUS_OPERATOR = "ose-prometheus-operator"
+MANAGED_STORAGECLUSTER_SIZE = "1"
```

Comment:
As discussed in https://chat.google.com/room/AAAASHA9vWs/dG9J9OB7Sas, we just need to confirm that ROSA CLI has the check that only 1 and 4 TB are supported and 1 TB is indeed the default. Not needed for merging this PR.. but we need to keep it in test
@suchita-g

Reply:
This will change based on configuration, right? So wouldn't it be better to not have it as a constant but take it from config files? As it is used as default vaule in [ocs_ci/utility/rosa.py](https://github.com/red-hat-storage/ocs-ci/pull/5589/files#diff-f9200133e61c199bf224f2ab7d5ef817f03e2d0455c3d686300fdaa378d162b3), it could be placed in a [default config](https://github.com/red-hat-storage/ocs-ci/blob/master/ocs_ci/framework/conf/default_config.yaml).

Reply:
```
ADD-ON PARAMETERS
- ID:             size
  Name:           Managed StorageCluster size
  Description:    The requested capacity, in bytes, of the storage cluster to be deployed.
  Type:           number
  Required:       yes
  Editable:       yes
  Default Value:  1099511627776
```

size is an optional parameter hence keeping it optional in our CI too. changing its usage definition in code